tests/unit/test_column_noise.py

```diff
@@ -200,11 +224,13 @@ def test_generate_typographical_errors(dummy_dataset, column):
 
 
 # TODO: refactor this into its own test parameterized by noise functions
-def _validate_seed_and_noise_data(func, column, config):
+def _validate_seed_and_noise_data(noise_type, column, config):
     """Confirms randomness stream behavior and returns the noised data"""
-    noised_data = func(column, config, RANDOMNESS0, f"test_{func.__name__}")
-    noised_data_same_seed = func(column, config, RANDOMNESS0, f"test_{func.__name__}")
-    noised_data_different_seed = func(column, config, RANDOMNESS1, f"test_{func.__name__}")
+    noised_data = noise_type(column, config, RANDOMNESS0, f"test_{noise_type.name}")
```

Comment:
wait...how is noise_type a callable at this point?

Reply:
noise_type is a ColumnNoiseType or a noise function.  We are carrrying along the args here and this is where we actually run the noise function.

Reply:
If you look in the places it gets called we are choosing which noise functions to run, with what data, and with what config and here is where we actually run it.

Reply:
This is the function you refactored since every noise function will need to do this.